### PR TITLE
Added possibility to pass command line arguments

### PR DESCRIPTION
**Pull request by bchalls**
Added possibility to pass command line arguments to allow for batch yaml file upload

### CODE REVIEW

File: batchUpload.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+subdomain=''
+email=''
+password=''
```
Comment (hugovk) [inline batchUpload.sh]: I can foresee someone editing this file and accidentally committing it exposing their details.

Perhaps call this file something like batchUploadExample.sh, ignore batchUpload.sh from Git, and instruct people to copy the former to the latter and edit that.

Comment (bchalls) [reply batchUpload.sh]: Makes sense. I have made the change and pushed it.

**Comment by zachfeldman** [comment]
Almost there @bchalls and thanks for the PR! Would you mind just editing the docs to reflect that you have to copy the example file and rename it for this to work? Then I'll merge. Thanks for the review @hugovk !

**Comment by zachfeldman** [comment]
Looking good, thanks @bchalls . Merging!

**Pull request merged**